lib/ansible/modules/cloud/amazon/ec2_instance_facts.py

```diff
@@ -20,6 +20,7 @@
 author:
   - Michael Schuett, @michaeljs1990
   - Rob White, @wimnat
+requirements: [ "boto3", "botocore" ]
```

Comment:
This will still say:

Requires:
* boto
* boto3
* botocore

As the requirements get added to the doc fragment list rather than replacing it

Reply:
Yes. That was my question on IRC. How do I remove boto? Which fragment is it coming from? `aws` or `ec2`? I am trying to locate those fragments.

Reply:
The requirements come from [aws doc fragment](../tree/devel/lib/ansible/utils/module_docs_fragments/aws.py)

I mention the problems that need addressing here in #30249

Reply:
I feel removing `boto` from `aws` fragment and adding individually to every module might be a better/safer option. Sure, it includes a lot of work. But we might get accurate information that way.

Reply:
For this particular issue, having a false requirement (boto) listed is still better than not having some real requirements not listed (boto3).